ui/components/data-sources/DataSourcesContainer.js

```diff
@@ -53,6 +53,10 @@ class DataSourcesContainer extends Component {
         this.setState({ showDeleteModal: true, selectedDataSource: dataSource });
     }
 
+    clearFilter() {
+        this.setState({ filter: {} });
```

Comment:
Why is filter an object?

Reply:
It was like that previously, someone probably thought that there might be filters by more criteria.

Reply:
It's true, my bad!